### DISCUSSION TITLE
Ente Auth - Minimize to Tray

### DISCUSSION THREAD

**Discussion by dudefoxlive** [ente-io/ente#3835] [Enhancements]
I know Ente Auth has an option to hide the window by right clicking the tray icon and choosing "Hide Window" but could we get an option in settings to automatically minimize to tray?

**Comment by birnam** [comment]
Actually even that option hasn't worked for the last couple of releases. Right clicking the Windows tray icon shows the 'Show Window / Hide Window" options but they don't DO anything.

edit: I suspect an upstream issue with the tray_manager dependency, there is a similar sounding issue open on that -- https://github.com/leanflutter/tray_manager/issues/53

**Reply by resuser** [reply]
Related discussions in #4360.

Personally, the show/hide options in the tray are working for me, but this functionality is cumbersome to use.
My ideal usage (Win 11) would be:
  - Ente Auth launches on boot hidden in the tray.
  - Clicking on the tray icon while hidden would show the window.
  - Clicking on the tray icon while active would hide the window.
  - Minimizing the window would also hide it.
